airbyte-integrations/connectors/destination-s3-glue/src/main/resources/spec.json

```diff
@@ -132,19 +132,70 @@
                 "description": "Whether the input json data should be normalized (flattened) in the output JSON Lines. Please refer to docs for details.",
                 "default": "Root level flattening",
                 "enum": ["No flattening", "Root level flattening"]
+              },
+              "stringify": {
+                "type": "string",
+                "title": "Stringify JSON",
+                "description": "Whether nested objects in the input json data should be converted to strings in the output JSON Lines. Please refer to docs for details.",
+                "default": "Default",
+                "enum": ["Default", "Stringify"]
               }
             }
           }
         ],
         "order": 5
       },
+      "numeric": {
+        "title": "Numeric Type",
```

Comment:
Should there be a default Numeric Type?

Reply:
I would set the default numeric type to be `double` to match the behavior of the Avro schema conversion in the S3 destination.